### PR TITLE
Chore(example) : customize page name

### PR DESCRIPTION
**Pull request by JulienIzz**
This PR adds some custom name for app pages in the menu, as well as new header description to describe what is displayed.

image

image

### CODE REVIEW

File: packages/example/src/components/VirtualizedSpatialGrid.tsx [hunk 1]
```diff
@@ -28,8 +28,8 @@ export const VirtualizedSpatialGrid = ({ containerStyle }: { containerStyle?: Vi
         data={hardcodedRabbitsArray}
         header={
           <Header
-            title="Rabbit Movies"
-            description="Delve into the delightful world of Rabbit Movies, where every film celebrates the charm and whimsy of our favorite fluffy friends. This category is a haven for rabbit lovers, featuring animated escapades and heartwarming family stories starring these adorable creatures."
+            title="Example of a virtualized grid with spatial navigation"
+            description="The grid shown on this page is virtualized, which means that when scrolling, the elements not shown in the screen are not rendered, improving performance. The items are also recycled when scrolling, which means that the same components are reused when scrolling, reducing the amount of memory used."
```
Comment (pierpo) [inline packages/example/src/components/VirtualizedSpatialGrid.tsx:32]:
```suggestion
            description="The grid shown on this page is virtualized, which means that when scrolling, the elements not shown in the screen are not rendered, improving performance. The items are also recycled when scrolling, which means that the same components are reused when scrolling, reducing the re-rendering time."
```

File: packages/example/src/components/Menu/MenuButton.tsx [hunk 2]
```diff
@@ -5,44 +5,49 @@ import {
...
-  },
-);
+const ButtonContent = forwardRef<
+  View,
+  { icon: React.ReactElement; isFocused: boolean; isMenuOpen: boolean }
+>((props, ref) => {
+  const { isFocused, icon, isMenuOpen } = props;
+  const anim = useFocusAnimation(isFocused && isMenuOpen);
+  const accessibilityProps = useSpatialNavigatorFocusableAccessibilityProps();
+  const Icon = React.cloneElement(icon, {
```
Comment (pierpo) [inline packages/example/src/components/Menu/MenuButton.tsx:26]: This shouldn't be necessary 🤔

Comment (JulienIzz) [reply packages/example/src/components/Menu/MenuButton.tsx]: Idem

File: packages/example/src/components/VirtualizedSpatialGrid.tsx (hunk 1 above)
Comment (JulienIzz) [reply packages/example/src/components/VirtualizedSpatialGrid.tsx]: Done 👍

**Review by pierpo — APPROVED** [review]

**Pull request merged**